hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -435,9 +450,7 @@ public OutputStream openFileForWrite(final Path path, final boolean overwrite) t
         client,
         AbfsHttpConstants.FORWARD_SLASH + getRelativePath(path),
         offset,
-        abfsConfiguration.getWriteBufferSize(),
-        abfsConfiguration.isFlushEnabled(),
-        abfsConfiguration.isOutputStreamFlushDisabled());
+            populateAbfsOutputStreamContext());
```

Comment:
Fix the indentation -- it seems like it's being changed from the previous format here (you're using 8spaces for parameters on new lines vs 4spaces previously)